src/components/signup.component.js

```diff
@@ -2,28 +2,36 @@ import { html, LitElement, css } from "lit-element";
 import "./button.component";
 import "./input.component";
 import { Router } from "@vaadin/router";
+import { postwithoutAuth } from "../services/api.services";
 class SignupComponent extends LitElement {
   constructor() {
     super();
-    this._api = "https://conduit.productionready.io/api";
 
-    this._userName = "";
-    this._email = "";
-    this._password = "";
+    this.userName = "";
+    this.email = "";
+    this.password = "";
 
     this.showError = false;
-    this._errors;
+    this.errors;
 
-    this._handleChange = this._handleChange.bind(this);
+    this.handleChange = this.handleChange.bind(this);
 
-    this._singUp = this._singUp.bind(this);
+    this.singUp = this.singUp.bind(this);
+    this.isToken = window.localStorage.getItem("token") ? true : false;
   }
   static get properties() {
     return {
       showError: { type: Boolean },
-      _errors: { type: Array }
+      errors: { type: Array }
     };
   }
+  connectedCallback()
+  {
+    if(this.isToken)
```

Comment:
What will happen in case there is no token?

Reply:
resolving as unnecessary condition not required